### PR TITLE
fix#661 - added updated screenshots of the entire app

### LINKED ISSUE #661: fix: Add screen shots of the entire app in Readme.md file
**Summary:**
The Readme of the app still misses a lot of screen shots of the app. 
We can show many of our fragments.
I want to add the rest of the pages in the screenshot section of Readme

### PR DESCRIPTION
**Pull request by devansh-299**
## Issue Fix
Fixes #661 

## Description
Updated the screenshots and added a link to `Screenshot.md` in `README.md`
For reviewing purposes checkout the `Screenshot.md` [here](https://github.com/devansh-299/mobile-wallet/blob/fix%23661/Screenshot.md)

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by devansh-299** [comment]
@rachittt please have a look

File: Screenshot.md [hunk 1]
```diff
@@ -1,35 +1,32 @@
...
-<img src="https://user-images.githubusercontent.com/37215508/76330738-3a7d3000-6314-11ea-9a4f-d6b04e433e6e.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76330810-4ec12d00-6314-11ea-8553-e7a8bf576a9a.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76331165-c3946700-6314-11ea-83fc-9e42a5553af3.jpg" width="288" height="500" />
-
+<img src="https://user-images.githubusercontent.com/46667021/82584052-8ddfea00-9bb1-11ea-92a9-8c83abf6322e.jpg" width="288" height="500" />
+<img src="https://user-images.githubusercontent.com/46667021/82584248-d8616680-9bb1-11ea-97b2-6a6e5129687a.jpg" width="288" height="500" />
+<img src="https://user-images.githubusercontent.com/46667021/82584330-faf37f80-9bb1-11ea-8443-bb676a99d714.jpg" width="288" height="500" />
+<img src="https://user-images.githubusercontent.com/46667021/82584418-18284e00-9bb2-11ea-94d2-5d9dedff8fb2.jpg" width="288" height="500" />
+<img src="https://user-images.githubusercontent.com/46667021/82584793-a13f8500-9bb2-11ea-847d-bb0e05279309.jpg" width="288" height="500" />
```
Comment (Bhavnaharitsa) [inline Screenshot.md]: Amazing organization of images. However, I feel the Markdown way is better. It helps you to organize images in tables, give a description for every image. You can refer to this: https://gist.github.com/grocky/5d75af69b4f4c10304bf. and also avoids the hardcoded way, which might differ when the orientation changes and the screen size changes. You might choose to ignore it as well :).

**Comment by luckyman20** [comment]
I would suggest adding labels and spacing to screenshots. @devansh-299

**Comment by devansh-299** [comment]
@Bhavnaharitsa thank you for the suggestion, I will update the PR accordingly. 
@luckyman20 should I label screens as for example - `Send Fragment` or by describing the process like "Making a Payment"

**Comment by luckyman20** [comment]
@devansh-299 well, you can do something like 'sending a payment' or 'transfer money screen' as it could be understood by both developers and users.

File: Screenshot.md [hunk 2]
```diff
@@ -1,35 +1,44 @@
...
+|Google Signup|Mobile Verification|Registration Screen|
+|------|------|------|
+|<img src="https://user-images.githubusercontent.com/46667021/82584418-18284e00-9bb2-11ea-94d2-5d9dedff8fb2.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82584793-a13f8500-9bb2-11ea-847d-bb0e05279309.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82584872-bd432680-9bb2-11ea-8a99-eca1a031db20.jpg" width="288" height="500" />|
+
+|Passcode Verification|Home Screen|Recent Transactions|
+|------|------|------|
+|<img src="https://user-images.githubusercontent.com/46667021/82588474-7b1ce380-9bb8-11ea-9a12-c72a57a7d5fe.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82584944-dba92200-9bb2-11ea-87ca-4f87b078d593.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82585044-ff6c6800-9bb2-11ea-86b2-764e7ec47689.jpg" width="288" height="500" />|
+
+
+|Payment Screen|Scanning QR Code for VPA Address|Making a Payment|
```
Comment (rachittt) [inline Screenshot.md:14]:
```suggestion
|Selecting Transfer Method|Scanning QR Code for VPA Address|Making a Payment|
```

Comment (rchtgpt) [inline Screenshot.md]:
```suggestion
|Selecting Transfer Method|Scanning QR Code for VPA Address|Making a Payment|
```

File: Screenshot.md [hunk 3]
```diff
@@ -1,35 +1,44 @@
...
+
+|Requesting Payment|Request Payment QR Code |Transaction History|
+|------|------|------|
+|<img src="https://user-images.githubusercontent.com/46667021/82585523-c2ed3c00-9bb3-11ea-999b-c68582f3f811.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82585345-7bff4680-9bb3-11ea-913a-9f2a27f9c83a.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82585589-dbf5ed00-9bb3-11ea-8f28-5ed2150e107d.jpg" width="288" height="500" />|
+
+|Transaction Details|Specification Transactions History|Transaction Receipt|
+|------|------|------|
+|<img src="https://user-images.githubusercontent.com/46667021/82585645-f8922500-9bb3-11ea-8a5e-6190d2acddde.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82585705-1790b700-9bb4-11ea-9f77-05057301a51d.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82585802-4313a180-9bb4-11ea-897a-b5da9a7ff0d9.jpg" width="288" height="500" />|
+
+|Bank Accounts Screen|Linking New Account|Phone Number Verification|
```
Comment (rchtgpt) [inline Screenshot.md]:
```suggestion
|Bank Accounts Screen|Linking Bank Account|Phone Number Verification|
```

File: Screenshot.md [hunk 4]
```diff
@@ -1,35 +1,44 @@
...
+|------|------|------|
+|<img src="https://user-images.githubusercontent.com/46667021/82585960-7e15d500-9bb4-11ea-88a0-e9f3def320cd.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82586461-33e12380-9bb5-11ea-9ca6-9fe6199a29bb.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82586565-6723b280-9bb5-11ea-961e-19872e7f70f8.jpg" width="288" height="500" />|
+
+
+|Merchants Screen|Merchant Payment and History Screen|User Profile Screen|
+|------|------|------|
+|<img src="https://user-images.githubusercontent.com/46667021/82587508-efef1e00-9bb6-11ea-81b2-54d9b8239ff6.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82587653-317fc900-9bb7-11ea-993b-36d558e2d499.jpg" width="288" height="500" />|<img src="https://user-images.githubusercontent.com/46667021/82586663-8fabac80-9bb5-11ea-8373-cd962931aa70.jpg" width="288" height="500" />|
+
+
+|Edit Profile Screen|Change Password Screen|Changing Passcode|
```
Comment (rchtgpt) [inline Screenshot.md]:
```suggestion
|Edit Profile Screen|Change Password Screen|Change Passcode Screen|
```

**Review by rchtgpt — CHANGES_REQUESTED** [review]

**Pull request merged**